woocommerce/woocommerce-admin
# Ignore refunded amounts when calculating gross total.

Fixes #2650 

Ignore negative values for `gross_total` when `SUM()`ing for Orders and Revenue reports.

### Screenshots

**Before**

<img width="883" alt="Screen Shot 2019-08-20 at 11 57 15 AM" src="https://user-images.githubusercontent.com/63922/63376017-b8aac280-c341-11e9-9a4b-6bb6af490720.png">

**After**

<img width="882" alt="Screen Shot 2019-08-20 at 11 56 35 AM" src="https://user-images.githubusercontent.com/63922/63376043-bfd1d080-c341-11e9-9ac5-164fe3d57da1.png">

### Detailed test instructions:

- Create a new order. Note the gross and net revenue amounts
- Go to the revenue report, set date range to "Today"
- Note the appropriate values in the Gross and Net summary numbers
- Return to the order detail screen and refund the order in full
- Go back to the revenue report, set date range to "Today"
- Note the the appropriate value appearing in the refunds summary number
- Note the appropriate net revenue value (should be zero)
- Note that Gross revenue **has not changed**



### Changelog Note:



Fix: gross order total calculation.
